src/CliqStateMachineUtils.jl

```diff
@@ -898,7 +898,8 @@ directPriorMsgIDs, directFrtlMsgIDs, directAssignmentIDs, mcmcIterationIDs
 function determineCliqVariableDownSequence(subfg::AbstractDFG, cliq::TreeClique; solvable::Int=1)
   frtl = getCliqFrontalVarIds(cliq)
 
-  adj = DFG.getAdjacencyMatrix(subfg, solvable=solvable)
+  #TODO don't use this getAdjacencyMatrixSymbols
+  adj = DFG.getAdjacencyMatrixSymbols(subfg, solvable=solvable)
```

Comment:
This has to be updated to use `getBiadjacencyMatrix`

Reply:
understood